qiita_pet/templates/list_analyses.html

```diff
@@ -22,83 +43,152 @@
+          $.get('/artifact/info/', {ids: JSON.parse(row.data()[0]), only_biom: 'False' })
+            .done(function ( data ) {
+              if (data['status']=='success') {
+                $('td', row.child()).html(format_biom_rows(data.data, row.index(), for_study_list=false)).show();
+              } else {
+                bootstrapAlert('ERROR: ' + data['msg'], "danger", 10000);
+              }
+            });
+        }
+    });
 
-      $('#analyses-table tbody').on('click', 'div.details-control', function () {
-          var table = $('#analyses-table').DataTable();
-          var tr = $(this).closest('tr');
-          var row = table.row( tr );
+    // connecting searches
+    $('#user-analyses-table').on('search.dt', function () {
+      var search_text = $('.dataTables_filter input').val();
+      $('#public-analyses-table').DataTable().search(search_text).draw();
+    });
 
-          if ( row.child.isShown() ) {
-            // This row is already open - close it
-            row.child.hide();
-            tr.removeClass('shown');
-          } else {
-            // Open this row
-            // modified from: https://jsfiddle.net/8rejaL88/2/
-            tr.addClass('shown');
-            row.child('<p><center><img src="{% raw qiita_config.portal_dir %}/static/img/waiting.gif" style="display:block;margin-left: auto;margin-right: auto"/></center></p>', 'no-padding' ).show();
-            $.get('/artifact/info/', {ids: JSON.parse(row.data()[0]), only_biom: 'False' })
-              .done(function ( data ) {
-                if (data['status']=='success') {
-                  $('td', row.child()).html(format_biom_rows(data.data, row.index(), for_study_list=false)).show();
-                } else {
-                  bootstrapAlert('ERROR: ' + data['msg'], "danger", 10000);
-                }
-              });
-          }
-      });
-    {% end %}
+    // connecting paging size
+    $('#user-analyses-table').on('length.dt', function (e, settings, len) {
+      $('#public-analyses-table').DataTable().page.len(len).draw();
+    });
   });
 </script>
 {% end %}
 
 {%block content %}
 
-<div id='no-analyses' class='jumbotron'>
-  <h1>No Analyses available</h1>
-  <h3><a href="{% raw qiita_config.portal_dir %}/study/list/">Create an analysis</a></h3>
-</div>
-
-{% if analyses %}
-  <table id="analyses-table" class="table table-bordered gray-msg">
-    <thead>
-      <tr>
-        <th>Artifacts</th>
-        <th>Analysis Name</th>
-        <th>Timestamp</th>
-        <th>Mapping File</th>
-        <th>Delete?</th>
-      </tr>
+<div class="row">
+  <div class="col-sm-12" id="user-studies-div">
+    <h3 class="gray-msg">Your Analyses</h3>
+    <table id="user-analyses-table" class="table table-bordered gray-msg">
+      <thead>
+        <tr>
+          <th>Artifacts</th>
+          <th>Analysis Name</th>
+          <th>Timestamp</th>
```

Comment:
I believe this is the creation timestamp. Could it instead read "Creation Timestamp" or something like that?